packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -135,6 +126,14 @@ export interface EditableTree extends Iterable<EditableField> {
      * when the fields are getting changed while iterating.
      */
     [Symbol.iterator](): IterableIterator<EditableField>;
+
+    /**
+     * Creates a new field at this node.
+     *
+     * Use {@link EditableField.insertNodes} to crete fields of kind `value` as currently
+     * it is not possible to have trees with already populated fields of this kind.
+     */
+    [createFieldSymbol](fieldKey: FieldKey, newContent: ITreeCursor): EditableField | undefined;
```

Comment:
Consider taking in `newContent: ITreeCursor[]` or `newContent: ITreeCursor | ITreeCursor[]` for the sake of initializing a field with multiple nodes.

Reply:
Fixed.